### PR TITLE
ER-866_Testing on Preprod.

### PR DESCRIPTION
**Pull request by shomavg**
This will be merged to master and then tested  on PreProd.

### CODE REVIEW

File: src/Provider/Provider.Web/Filters/CheckProviderBlockedFilter.cs [hunk 1]
```diff
@@ -28,7 +28,7 @@ public CheckProviderBlockedFilter(IConfiguration configuration)
 
                 var ukprnFromUrl = context.RouteData.Values[RouteValues.Ukprn].ToString().ToUpper();
 
-                if (blockedProviderAccountIds.Contains(ukprnFromUrl))
+                if (!blockedProviderAccountIds.Contains(ukprnFromUrl))
```
Comment (chrisjensenuk) [inline src/Provider/Provider.Web/Filters/CheckProviderBlockedFilter.cs]: Now this is a whitelist can we rename this to `whitelistedProviderAccountIds`

File: src/Provider/Provider.Web/appsettings.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ﻿{
-  "WhitelistedProvidersList": "",
+  "WhitelistedProvidersList": "10007289;10033670;10001191;10000673;10004736",
```
Comment (sachpatel) [inline src/Provider/Provider.Web/appsettings.json]: `ListOfWhitelistedProvidersList`

Comment (sachpatel) [reply src/Provider/Provider.Web/appsettings.json]: why are these hardcoded, they should go in via release variables

File: src/Shared/Recruit.Shared.Web/Configuration/EnvironmentNames.cs [hunk 3]
```diff
@@ -12,7 +12,7 @@ public static class EnvironmentNames
         public const string PREPROD = "PREPROD";
         public const string PROD = "PROD";
 
-        public static string[] GetTestEnvironmentNames() => new []{ Development, AT, TEST, DEMO };
+        public static string[] GetTestEnvironmentNames() => new []{ Development, AT, DEMO };
```
Comment (sachpatel) [inline src/Shared/Recruit.Shared.Web/Configuration/EnvironmentNames.cs]: TEST is a test environment

File: src/Provider/Provider.Web/Filters/CheckProviderBlockedFilter.cs (hunk 1 above)
Comment (sachpatel) [reply src/Provider/Provider.Web/Filters/CheckProviderBlockedFilter.cs]: they are not account id's, they are ukprns

File: src/Shared/Recruit.Shared.Web/Configuration/EnvironmentNames.cs (hunk 3 above)
Comment (shomavg) [reply src/Shared/Recruit.Shared.Web/Configuration/EnvironmentNames.cs]: its just for testing. It will be put back later.

Comment (sachpatel) [reply src/Shared/Recruit.Shared.Web/Configuration/EnvironmentNames.cs]: why can't this be tested on PREPROD then. It's what we did for testing blocked employers. We shouldn't be making temporary code changes that go into master

File: src/Provider/Provider.Web/appsettings.json (hunk 2 above)
Comment (sachpatel) [reply src/Provider/Provider.Web/appsettings.json]: WhitelistedProviders would do

Comment (sachpatel) [reply src/Provider/Provider.Web/appsettings.json]: also I don't see any ARM template changes so how will this work

File: src/Shared/Recruit.Shared.Web/Configuration/EnvironmentNames.cs (hunk 3 above)
Comment (shomavg) [reply src/Shared/Recruit.Shared.Web/Configuration/EnvironmentNames.cs]: i will not be merging into master for now ... once testing is passed... i ll revert the changes that were environment specific.

**Pull request merged**